mondrian/src/main/java/mondrian/rolap/RolapLevel.java

```diff
@@ -412,7 +412,9 @@ private Property lookupProperty(List<Property> list, String propertyName) {
         } else if (type.equals("Numeric")) {
             return Property.Datatype.TYPE_NUMERIC;
         } else if (type.equals("Integer")) {
-            return Property.Datatype.TYPE_NUMERIC;
```

Comment:
We should probably add a parameter for this. I think the new behavior is better, but some people may have been relying on the old Integer->Numeric mapping. Side effects of this may be, for example, that reports which displayed numbers in the 0.00 format would change to 0. People using a custom member formatter may also encounter class cast exceptions.

You can create a new Mondrian property by adding it to